Test Failure seems unrelated. This is my first PR, so wanted to make sure, if I understand it correctly.

Seeing Error on - `TestSeriesReductions.test_empty_multi`

`AssertionError: Caused unexpected warning(s): [('ResourceWarning', ResourceWarning('unclosed transport <asyncio.sslproto._SSLProtocolTransport object at 0x000001B4D3B9EE20>'), 'C:\\Miniconda\\envs\\pandas-dev\\lib\\asyncio\\sslproto.py', 321)]`

Also got a pop up on the Azure Pipeline that this test has failed `124` times in the past `14` days.


[Test Failure](https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=58341&view=logs&j=69addb8c-d01a-5d3b-91bf-b74d46cb8150&t=cd1a349e-a8c9-5d71-a820-b09fec3303fd&l=389)

--- a/pandas/core/tools/datetimes.py
+++ b/pandas/core/tools/datetimes.py
@@ -862,8 +862,8 @@ dtype='datetime64[ns]', freq=None)
 
     >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'],
     ...                utc=True)
-    DatetimeIndex(['2018-10-26 17:30:00+00:00', '2018-10-26 17:00:00+00:00'], \
-dtype='datetime64[ns, UTC]', freq=None)
+    DatetimeIndex(['2018-10-26 17:30:00+00:00', '2018-10-26 17:00:00+00:00'],
+                  dtype='datetime64[ns, UTC]', freq=None)
     """
     if arg is None:
         return None

--- a/pandas/core/tools/datetimes.py
+++ b/pandas/core/tools/datetimes.py
@@ -787,8 +787,7 @@ def to_datetime(
         array/Series).
 
         In case input is list-like and the elements of input are of mixed
-        timezones, return will have object type Index if utc parameter is not
-        passed in as True.
+        timezones, return will have object type Index if utc=False.
 
     See Also
     --------
@@ -859,11 +858,15 @@ def to_datetime(
 dtype='datetime64[ns]', freq=None)
 
     In case input is list-like and the elements of input are of mixed
-    timezones, return will have object type Index if utc parameter is not
-    passed in as True.
+    timezones, return will have object type Index if utc=False.
 
     >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'])
     Index([2018-10-26 12:00:00-05:30, 2018-10-26 12:00:00-05:00], dtype='object')
+
+    >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'],
+    ...                utc=True)
+    DatetimeIndex(['2018-10-26 17:30:00+00:00', '2018-10-26 17:00:00+00:00'], \
+dtype='datetime64[ns, UTC]', freq=None)
     """
     if arg is None:
         return None

--- a/pandas/core/tools/datetimes.py
+++ b/pandas/core/tools/datetimes.py
@@ -786,9 +786,6 @@ def to_datetime(
         return will have datetime.datetime type (or corresponding
         array/Series).
 
-        In case input is list-like and the elements of input are of mixed
-        timezones, return will have object type Index if utc=False.
-
     See Also
     --------
     DataFrame.astype : Cast argument to a specified dtype.

